src/main/java/store/model/Sale.java

```diff
@@ -13,10 +14,11 @@
 @Entity
 @Table(name = "sale")
 @Data
+@NoArgsConstructor
```

Comment:
You don't need to use @NoArgsConstructor. There aren't other constructors with parameters